configuration/openmrs/apps/reports/reports.json

```diff
@@ -0,0 +1,88 @@
+{
+  "visitReport10": {
+    "name": "Visit Report",
```

Comment:
This suggests the name may not be i18n-able. Do you confirm?

Reply:
Yes, report names are not translatable at the moment. There is a missing` | translate` in [bahmniapps](https://github.com/Bahmni/openmrs-module-bahmniapps/blob/master/ui/app/reports/views/reports.html#L36).
I verified translating the report names by adding it to bahmniapps in my vagrant and it worked. Maybe this should be added into bahmniapps master.

Reply:
OK. That would be great yes.
Could you create the PR to Bahmni Apps?

Reply:
https://github.com/Bahmni/openmrs-module-bahmniapps/pull/295

Reply:
Thanks, I'll merge that Bahmni Apps PR.
In the mean time, let's add the Reports as is.